docs/overview.md

```diff
@@ -13,7 +13,63 @@ nav_order: 2
 ---
 ## Overview
 Inferno is framework for creating, executing, and sharing tests for health IT
-systems providing standardized FHIR APIs.
+systems providing standardized FHIR APIs.  You use Inferno to create
+**Inferno Test Kits** which are standalone, portable test applications that
+are tailored to test specific FHIR-enabled data exchange use cases as described in [FHIR Implementation Guides](http://fhir.org/guides/registry/).
+
+Examples of test applications (Test Kits) built using Inferno include:
+* The [US Core Test Kit](https://github.com/inferno-framework/us-core-test-kit) provides tests for servers implementing [US Core Implementation Guide](http://hl7.org/fhir/us/core/).
```

Comment:
These bullets aren't wrapped.

Reply:
Ah, I was hoping you wouldn't notice, my tool is really bad at wrapping markdown bullets :)  Thanks for keeping me honest!